include/openssl/configuration.h.in

```diff
@@ -79,8 +79,8 @@ extern "C" {
 # ifndef OPENSSL_NO_EGD
 #  define OPENSSL_NO_EGD
 # endif
-# ifndef CRYPTO_NO_MDEBUG
-#  define CRYPTO_NO_MDEBUG
+# ifndef OPENSSL_NO_MDEBUG
+#  define OPENSSL_NO_MDEBUG
```

Comment:
I missed that this was wrong. It still is. It should be `OPENSSL_NO_CRYPTO_MDEBUG`. My nit was about the missing indent in the quic stanza below.

Reply:
I will not do commits while on the phone.  I will not do commits while on the phone... sigh.  sorry for wasting your time.  fixup pushed.